internal/pkg/vm/internal/vmcontext/invocation_context.go

```diff
@@ -317,10 +317,15 @@ func (ctx *invocationContext) Send(toAddr address.Address, methodNum abi.MethodN
 		}
 	}()
 
+	// DRAGONS: remove these once specs actors project enforces Send params
```

Comment:
FYI This will go the other way, with nil being the correct value for "no params"